core/server/common/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -91,112 +95,73 @@ protected AbstractMaster(Journal journal, Clock clock,
   }
 
   @Override
-  public void processJournalCheckpoint(JournalInputStream inputStream) throws IOException {
-    JournalEntry entry;
-    try {
-      while ((entry = inputStream.read()) != null) {
-        processJournalEntry(entry);
-      }
-    } finally {
-      inputStream.close();
-    }
-  }
-
-  @Override
-  public void start(boolean isLeader) throws IOException {
+  public void start(boolean isPrimary) throws IOException {
     Preconditions.checkState(mExecutorService == null);
     mExecutorService = mExecutorServiceFactory.create();
-    mIsLeader = isLeader;
-    LOG.info("{}: Starting {} master.", getName(), mIsLeader ? "leader" : "standby");
-    if (mIsLeader) {
-      Preconditions.checkState(mJournal instanceof MutableJournal);
-      mJournalWriter = ((MutableJournal) mJournal).getWriter();
-
+    mIsPrimary = isPrimary;
+    LOG.info("{}: Starting {} master.", getName(), mIsPrimary ? "primary" : "secondary");
```

Comment:
I would prefer this to be two different log statements in the appropriate if blocks.